forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -83,17 +87,15 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * Checks the balance of a given PaymentMethod using a ForagePINEditText
-     *
-     * @param params The parameters required for balance inquiries, including
-     * a reference to a ForagePINEditText and PaymentMethod ref
+     * Checks the balance of an EBT Card via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element.
      *
-     * @return A ForageAPIResponse indicating the success or failure of the operation.
-     * On success, returns an object with `snap` (SNAP) and `cash` (EBT Cash) fields, whose values
-     * indicate the current balance of each respective tender
-     *
-     * @throws ForageConfigNotSetException If the passed ForagePINEditText instance
-     * hasn't had its ForageConfig set via .setForageConfig().
+     * @param CheckBalanceParams A model that passes a [`foragePinEditText`][com.joinforage.forage.android.ui.ForagePINEditText] instance and a `paymentMethodRef`, found in the response from a call to [tokenizeEBTCard] or the [Create a `PaymentMethod`](https://docs.joinforage.app/reference/create-payment-method) endpoint, as the [CheckBalanceParams] that Forage uses to check the balance of an EBT Card. 
+     * @return A [ForageApiResponse] object. 
+     * 
+     * On success, the object includes `snap` and `cash` fields that indicate the EBT Card's current SNAP and EBT Cash balances.  
+     * 
+     * On failure, the response includes a list of [ForageError][com.joinforage.forage.android.network.model.ForageError] objects that you can unpack to troubleshoot the issue. 
```

Comment:
Also optional:

Maybe worth referencing the most relevant SDK error handling document in our public docs site here?

And maybe worth linking to the "Test EBT Cards" section [for Balance inquiry exceptions](https://docs.joinforage.app/docs/test-ebt-cards#balance-inquiry-exceptions)?

Reply:
@djoksimo I will follow up in Linear!